system/spd.h

```diff
@@ -0,0 +1,47 @@
+#ifndef _SPD_H_
+#define _SPD_H_
+/**
+ * SPDX-License-Identifier: GPL-2.0
+ *
+ * \file
+ *
+ * Provides access to SPD parsing and printing functions.
+ *
+ * Copyright (C) 2004-2023 Sam Demeulemeester.
+ */
+
+/** Rounding factors for timing computation
+ *
+ *  These factors are used as a configurable CEIL() function
+ *  to get the upper int from a float past a specific decimal point.
+ */
+
+#define DDR5_ROUNDING_FACTOR    30
+#define ROUNDING_FACTOR         0.9f
+
```

Comment:
These two definitions are only used inside `spd.c` so are not part of the unit API and should be moved into `spd.c`.

Reply:
Good idea, done.